features/tests/factories.py

```diff
@@ -67,3 +76,22 @@ class Meta:
     municipality = factory.Faker("city", locale="fi_FI")
     phone_number = factory.Faker("phone_number")
     email = factory.Faker("company_email")
+
+
+class OpeningHoursPeriodFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = OpeningHoursPeriod
+
+    feature = factory.SubFactory(FeatureFactory)
+    valid_from = factory.Faker("date_between", start_date="-1y", end_date="today")
+    valid_to = factory.Faker("date_between", start_date="today", end_date="+1y")
+
+
+class OpeningHoursFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = OpeningHours
+
+    period = factory.SubFactory(OpeningHoursPeriodFactory)
+    day = factory.Faker("pyint", min_value=1, max_value=7)
+    opens = factory.Faker("time_object")
+    closes = factory.Faker("time_object")
```

Comment:
will it cause any issues if `Faker` generates `closes` time, which would be before `opens` time?

Reply:
you could do some `LazyFunction` which would look at the `opens` time maybe?

not sure how critical all of this is anyway 😅

Reply:
I'd say that for the time being it might not be critical since this structure might change in the future still. Also we have not yet fully decided how we should store the information that a place is open 18-02. It could be just one `OpeningHours` or two (18-24, 00-02).

Reply:
yeah, sure, fair enough 🙂